src/core/qgsmessagelog.h

```diff
@@ -71,7 +71,7 @@ class CORE_EXPORT QgsMessageLog : public QObject
      * This signal is emitted for all messages received by the log, regardless of the \a notifyUser flag's
      * value for the message.
      */
-    void messageReceived( const QString &message, const QString &tag, Qgis::MessageLevel level );
+    void messageReceived( const QString &message, const QString &tag, Qgis::MessageLevel level, Qgis::StringFormat type );
```

Comment:
Unfortunately I missed this in the earlier review, but adding the new argument to this signal breaks stable API.

We'll need to add a new signal with the new signature, so that existing connections in plugins continue to work.

Maybe something like `messageReceivedWithType`. 

Then whenever we emit the existing messageReceived signal, we'll also emit the new messageReceivedWithType signal. 

Optionally the old signal could be marked as deprecated with the Q_DECL_DEPRECATED / SIP_DEPRECATED macros